Tests/ValetIntegrationTests/ValetIntegrationTests.swift

```diff
@@ -563,8 +601,7 @@ class ValetIntegrationTests: XCTestCase
         XCTAssertEqual(noItemsFoundError, anotherFlavor.migrateObjects(matching: valet.keychainQuery, removeOnCompletion: true))
     }
 
-    // FIXME: Looks to me like this test may no longer be valid, need to dig a bit
```

Comment:
After some investigation, it seem like iOS 11 and iOS 12 only return synchronizable (iCloud) keychain items if `kSecAttrSynchronizable` is specified in the query (this is unlike iOS 13, and I believe – but I haven't verified – iOS 9 and iOS 10).

therefore, this test would fail, because the value set in `anotherFlavor` wouldn't be returned from the `conflictingQuery`. To fix the test, I've used a Valet with a differing identifier rather than a different flavor.